scale/docs/rest/workspace.rst

```diff
@@ -2,181 +2,328 @@
+|                "is_active": true,                                                                                       |
+|                "used_size": 0,                                                                                          |
+|                "total_size": 0,                                                                                         |
+|                "created": "2015-10-05T21:26:04.855Z",                                                                   |
+|                "archived": null,                                                                                        |
+|                "last_modified": "2015-10-05T21:26:04.855Z"                                                              |
+|            },                                                                                                           |
+|            ...                                                                                                          |
+|        ]                                                                                                                |
+|    }                                                                                                                    |
++-------------------------------------------------------------------------------------------------------------------------+
+
+.. _rest_workspace_create:
+
++-------------------------------------------------------------------------------------------------------------------------+
+| **Create Workspace**                                                                                                    |
++=========================================================================================================================+
+| Creates a new workspace with associated configuration                                                                   |
++-------------------------------------------------------------------------------------------------------------------------+
+| **POST** /workspaces/                                                                                                   |
++-------------------------+-----------------------------------------------------------------------------------------------+
+| **Content Type**        | *application/json*                                                                            |
++-------------------------+-----------------------------------------------------------------------------------------------+
+| **JSON Fields**                                                                                                         |
++-------------------------+-------------------+----------+----------------------------------------------------------------+
+| name                    | String            | Required | The stable name of the workspace used for queries.             |
```

Comment:
It's not entirely obvious what "stable name" is here. Maybe something like "identifying name" or something?